extensions/data-workspace/src/dataworkspace.d.ts

```diff
@@ -69,6 +69,11 @@ declare module 'dataworkspace' {
 		 * Gets the supported project types
 		 */
 		readonly supportedProjectTypes: IProjectType[];
+
+		/**
+		 * Gets the project actions to be placed on the dashboard toolbar
+		 */
+		getProjectToolbarActions(): IProjectAction[];
```

Comment:
let's make this a readonly property. similar to supportedProjectTypes
and rename it to projectActions

Reply:
Updated the code to make projectActions a readonly property.